source/MapPlanetCard.h

```diff
@@ -60,6 +61,9 @@ class MapPlanetCard {
 
 	void Select(bool select = true);
 
+	static double Height();
+
+	static void ResetSize();
 
```

Comment:
Maintain two lines of whitespace between the last function in `public:` and `protected:`
```suggestion


```